### PR TITLE
Fixes #1033 loklak_server README.md upd, useful links added

### LINKED ISSUE #1033: Write an overview of loklak projects for about page
We have a number of loklak projects. The current about page does not cover all of these projects yet. Write a concise overview of the projects.

### PR DESCRIPTION
**Pull request by mad3E7cat**
### Short description
I have:
- [x] Corresponding issue: https://github.com/loklak/loklak_server/issues/1033
- [ ] Mentioned the Issue number in the pull request commit message `Fixes #1033 loklak_server README.md upd, useful links added`
- [x] There is strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Nice that you added links. What to you think about changing the syntax and marking text as a link?

`[description](https://link)` [description](https://link)  
instead of  
`description(https://link)` description(https://link)

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you like to be anonymous when searching things, want to archive tweets or mes
 - share tweets and tweet archives with other loklak users
 - search anonymously on your own search portal
 - create your own tweet search portal or statistical evaluations
-- use [Kibana](https://github.com/elastic/kibana) to analyze large amounts of tweets for statistical data.
+- use [Kibana](https://github.com/elastic/kibana) to analyze large amounts of tweets for statistical data(http://blog.loklak.net/tweet-analytics-with-loklak-and-kibana-as-a-search-front-end/).
```
Comment (yukiisbored) [inline README.md]: Please use the link syntax properly.

File: README.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ Run elasticsearch-head with:
 ## How do I configure loklak?
 
 The basis configuration file is in ```conf/config.properties```. To customize these settings place a file ```customized_config.properties``` to the path ```data/settings/```
+See documentation(http://dev.loklak.org/) for details.
```
Comment (yukiisbored) [inline README.md]: Please use the link syntax properly

File: README.md [hunk 3]
```diff
@@ -149,7 +150,7 @@ Hey, this is the tool for that! Just put http://loklak.org/api/search.rss?q=%23l
 
 ## What is the software license?
 
-LGPL 2.1
+LGPL 2.1 (https://www.gnu.org/licenses/old-licenses/lgpl-2.1)
```
Comment (yukiisbored) [inline README.md]: Please use the link syntax properly

**Comment by mad3E7cat** [comment]
Thanks for the reply!

**Comment by yukiisbored** [comment]
Please do the same to README.rst as well.

**Pull request merged**